lib/install/config/webpack/paths.yml

```diff
@@ -0,0 +1,7 @@
+# Restart webpack-watcher or webpack-dev-server if you make changes here
+paths:
+  src_path: app/javascript
+  config_path: config/webpack
+  node_modules_path: node_modules
+  dist_dir: packs
+  dist_path: public/packs
```

Comment:
@javan Agree, makes sense to remove `_path` suffix as we are accessing it through `paths`, however not sure about consolidating options. Basically,

```js
dist_dir === packs_dir // also can be called src_dir/entries_dir/output_dir 
```

Keeping `dist_dir` as a separate option offers flexibility when it comes to changes and also resolving paths isn't difficult. For ex - https://github.com/rails/webpacker/pull/153/files#diff-fe01b8431201c036acc33f2bcdccb77eR12 (`publicPath` resolution will fail due to this change).

```js
const publicPath = env.NODE_ENV !== 'production' && dev_server.enabled ?
  `http://${dev_server.host}:${dev_server.port}/` : `/${paths.dist_dir}/`
```

If a better name makes sense then we can do that, but would be ideal to keep it separate. 

@justin808 I can do that.